src/pcapi/routes/serialization/venues_serialize.py

```diff
@@ -11,6 +14,62 @@
 from pcapi.utils.date import format_into_utc_date
 
 
+MAX_LONGITUDE = 180
+MAX_LATITUDE = 90
+
+
+class PostVenueBodyModel(BaseModel):
+    address: str
+    bookingEmail: str
+    city: str
+    comment: Optional[str]
+    latitude: float
+    longitude: float
+    managingOffererId: str
+    name: str
+    publicName: Optional[str]
+    postalCode: str
+    siret: str
+    venueLabelId: Optional[str]
+    venueTypeId: Optional[str]
+
+    @validator("latitude", pre=True)
+    def validate_latitude(cls, raw_latitude):  # pylint: disable=no-self-argument
+        try:
+            latitude = Decimal(raw_latitude)
+        except InvalidOperation:
+            raise ValueError("Format incorrect")
+        else:
+            if latitude > MAX_LATITUDE or latitude < -MAX_LATITUDE:
```

Comment:
(détail / style)

```suggestion
            if not -MAX_LATITUDE < latitude < MAX_LATITUDE:
```

Idem pour la longitude plus bas.

C'est une suggestion de présentation. À n'appliquer que si c'est plus lisible pour toi. Si non, tu peux très bien garder ta version. :)